submission.json

```diff
@@ -899,7 +899,24 @@
       "callforpaper": true,
       "twitter": "@chicagoroboto",
       "topics": [ "android" ]
+    },
+    {
+      "title": "Android Makers FR",
+      "year": 2018,
+      "startdate": "2018/04/23",
+      "enddate": "2018/04/24",
+      "where": "Le Beffroi 2 Place Emile Cresp, Montrouge",
+      "homepage": "http://androidmakers.fr/",
+      "country": "France",
+      "city": "Paris",
+      "callforpaper": true,
+      "emojiflag": "🇫🇷"
```

Comment:
this line is missing a comma feel free to remove that, emoji flag is auto generated